affirm/src/main/java/com/affirm/android/Affirm.java

```diff
@@ -680,8 +699,8 @@ public static void startCheckout(@NonNull Activity activity, @NonNull Checkout c
         AffirmUtils.requireNonNull(activity, "activity cannot be null");
         AffirmUtils.requireNonNull(checkout, "checkout cannot be null");
         if (useVCN) {
-            VcnCheckoutActivity.startActivity(activity, vcnCheckoutRequest, checkout, caas,
-                    cardAuthWindow, receiveReasonCodes);
+            startVcnCheckout(activity, checkout, caas, null, false,
```

Comment:
@wangdaliu this change was overwritten when trying to fix merge conflicts, can you check this line in master and updated it if necessary?